doc/source/methods/KernelSHAP.ipynb

```diff
@@ -569,10 +569,10 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Consider a model $f$ that takes as an input $M$ features and an explanation model $g$. Assume that we want to explain the output of the model $f$ when applied to an input $x$. In reality, we may explain the change of the model output, $f(x)$ with respect to the average output over a set of records, $\\mathcal{D}$. Since the model is additive we can write\n",
+    "Consider a model $f$ that takes as an input $M$ features. Assume that we want to explain the output of the model $f$ when applied to an input $x$. In reality, the model output scale does not have a true zero, we may explain the change of the model output, $f(x)$ with respect to the average output over a set of records, $\\mathcal{D}$. For the explanation to be accurate, one requires\n",
```

Comment:
"In reality, the model output scale does not have a true zero,..." - this sentence is quite confusing, what is a "true zero"? Is there a better way to introduce the concept that we want to assign attributions to some function f(x) - E(f(x)) with respect to some distribution? I see earlier you introduce `base_rate`, maybe this can be expanded upon and defined?